Library/Homebrew/env_config.rb

```diff
@@ -29,10 +29,6 @@ module EnvConfig
         description: "If set, use `bat` for the `brew cat` command.",
         boolean:     true,
       },
-      HOMEBREW_BAT_CONFIG_PATH:                   {
-        description:  "Use this as the `bat` configuration file.",
-        default_text: "`$HOME/.bat/config`.",
```

Comment:
We don't want to remove this -- this will cause errors for anyone who has `HOMEBREW_BAT` set.

Instead, we should just update the `default_text` to reflect the current default config location.